src/app/screens/sign-up/sign-up.component.ts

```diff
@@ -0,0 +1,25 @@
+import { Component, OnInit } from '@angular/core';
+import { FormBuilder } from '@angular/forms';
+import { SignUp } from './interfaces/sign-up.interface';
+
+@Component({
+  selector: 'app-sign-up',
+  templateUrl: './sign-up.component.html',
+  styleUrls: ['./sign-up.component.scss'],
+  providers: [FormBuilder]
```

Comment:
why do you provide formBuilder?

Reply:
Error resolved when adding formBuilder as providers, but for now the class is not used, I will remove it.